gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -224,13 +224,155 @@ Thefore there is no need to repeat the KHR structs
   @unused RenderPassVersion                  Version
 }
 
-sub AttachmentReference convertAttachment(VkAttachmentReference attachment) {
-  return AttachmentReference(
-    Attachment: attachment.attachment,
-    Layout: attachment.layout,
-    AspectMask: 0, // This will be updated by VkRenderPassInputAttachmentAspectCreateInfo
+sub AttachmentDescription createAttachmentDescriptionObject(VkAttachmentDescription attachmentDescription) {
+ internalAttachmentDescription := AttachmentDescription(
+    Flags: attachmentDescription.flags,
+    Format: attachmentDescription.format,
+    Samples: attachmentDescription.samples,
+    LoadOp: attachmentDescription.loadOp,
+    StoreOp: attachmentDescription.storeOp,
+    StencilLoadOp: attachmentDescription.stencilLoadOp,
+    StencilStoreOp: attachmentDescription.stencilStoreOp,
+    InitialLayout: attachmentDescription.initialLayout,
+    FinalLayout: attachmentDescription.finalLayout,
     StencilLayout: null,
   )
+  return internalAttachmentDescription
+}
+
+sub AttachmentDescription createAttachmentDescription2Object(VkAttachmentDescription2 attachmentDescription2) {
```

Comment:
Nit: I think this naming conveys a message that we have `AttachmentDescription2` object internally. Normally, I wouldn't ask for a change but as we probably need to add one more function, renaming this is not much of an extra work. Maybe a better name could be `createAttachmentDescriptionObjectFrom2` and  `createAttachmentDescriptionObject`/ `createAttachmentDescriptionObjectFromOld`

Reply:
Done!

Reply:
Thanks!